changelog/2871.breaking.rst

```diff
@@ -0,0 +1,4 @@
+The property `~plasmapy.plasma.grids.AbstractGrid.recognized_quantities` is now a class method
+instead of a class property, as using ``classmethod`` and ``property`` decorators together is no longer
+allowed in Python 3.13. The syntax for accessing this dictionary has therefore changed
+from ``AbstractGrid.recognized_quantities`` to ``AbstractGrid.recognized_quantities()``.
```

Comment:
```suggestion
The property `~plasmapy.plasma.grids.AbstractGrid.recognized_quantities` of `~plasmapy.plasma.grids.AbstractGrid` is now a class method
instead of a class property, as using `classmethod` and `property` decorators together is no longer
allowed in Python 3.13. The syntax for accessing this dictionary has therefore changed
from :py:`AbstractGrid.recognized_quantities`` to :py:`AbstractGrid.recognized_quantities()`.
```